pkg/reconciler/domainmapping/reconciler.go

```diff
@@ -342,7 +342,7 @@ func (r *Reconciler) reconcileDomainClaim(ctx context.Context, dm *v1alpha1.Doma
 func (r *Reconciler) createDomainClaim(ctx context.Context, dm *v1alpha1.DomainMapping) error {
 	if !config.FromContext(ctx).Network.AutocreateClusterDomainClaims {
 		dm.Status.MarkDomainClaimNotOwned()
-		return fmt.Errorf("namespace %q does not own ClusterDomainClaim for %q", dm.Namespace, dm.Name)
+		return fmt.Errorf("no ClusterDomainClaim found for domain %q", dm.Name)
```

Comment:
nit: do we want to print out the namespace of domainmapping?

Reply:
I _think_ the namespace is irrelevant in this case since we didn't find the CDC at the cluster level, so there isn't any diagnostic purpose in having the namespace of the (at this point unrelated) domain mapping in the message. We don't include the namespace in e.g. [the error message when we get an error looking up the CDC](https://github.com/knative/serving/blob/e173fa0a61f255eeab852fbe21731ceac0f11c69/pkg/reconciler/domainmapping/reconciler.go#L328).